ui/src/table.test.tsx

```diff
@@ -178,7 +183,17 @@ describe('Table.tsx', () => {
       expect(wave.args[name]).toMatchObject(['rowname1', 'rowname2'])
     })
 
-    it('Fires event - single selection', () => {
+    it('Sets args - single selection', () => {
+      const { getAllByRole } = render(<XTable model={{ ...tableProps, isSingle: true }} />)
+      const radioButtons = getAllByRole('radio')
+
+      fireEvent.click(radioButtons[0])
+      fireEvent.click(radioButtons[1])
+
+      expect(wave.args[name]).toMatchObject(['rowname2'])
```

Comment:
let's make this more clear
```suggestion
      fireEvent.click(radioButtons[0])
      expect(wave.args[name]).toMatchObject(['rowname1'])
      fireEvent.click(radioButtons[1])

      expect(wave.args[name]).toMatchObject(['rowname2'])
```